lib/docker-compose/models/compose_container.rb

```diff
@@ -150,6 +150,27 @@ def prepare_volumes
     volumes
   end
 
+  #
+  # Prepare Hostconfig Bind mounts by converting
+  # relative paths into absolute paths
+  #
+  def prepare_volume_binds
```

Comment:
Can you please add/adjust the specs to cover this fix?
Since the tests are passing without this fix, seems like we have an uncovered case.